packages/babel-preset-env/scripts/build-data.js

```diff
@@ -186,20 +181,7 @@ const getLowestImplementedVersion = ({ features }, env) => {
     }, []);
 
   const unreleasedLabelForEnv = unreleasedLabels[env];
-  const envTests = tests.map(({ res: test, isBuiltIn }, i) => {
-    // Babel itself doesn't implement the feature correctly,
-    // don't count against it
-    // only doing this for built-ins atm
-    //
-    // NOTE: when/if compat-table adds a babel7 key, we'll want to update this
-    if (!test.babel6corejs2 && isBuiltIn) {
```

Comment:
Why is this check removed?

Reply:
The main reason for removing this check was that this script will not generate correct output before adding unstable `core-js` version results to `compat-table`. Since now it's used only for generating data for `core-js@2`, it's not so actual. However, some features can't be fully implemented and with this check, polyfills for them would be ignored. So I don't see any reason leave this additional check for built-ins which anyway ignored for syntax.

Reply:
Ok :+1: